_config.yml

```diff
@@ -48,6 +48,8 @@ requires_5_1v: "> **Note**: This feature is only available for TinyMCE 5.1 and l
 requires_5_2v: "> **Note**: This feature is only available for TinyMCE 5.2 and later."
 requires_5_3v: "> **Note**: This feature is only available for TinyMCE 5.3 and later."
 
+requires_jsscwar_230v: "> **Note**: This feature is only available for TinyMCE self-hosted server-side components version 2.3.0 and later."
```

Comment:
I guess we don't really care about hex as it's just an HCL bundle, but it's 2.4.0 for hex. Do you feel this wording is making it obvious that it's talking about each individual service war? e.g. `ephox-spelling`. Should we put it in a note of how to check your version?

Reply:
> I guess we don't really care about hex as it's just an HCL bundle

HCL bundle has it's own docs elsewhere.

> Do you feel this wording is making it obvious that it's talking about each individual service war? e.g. ephox-spelling

only way to be 100% clear is to spell out each one explicitly. I'd rather not.

> Should we put it in a note of how to check your version?

Updated to:
```
Note: This feature is only available for TinyMCE self-hosted server-side components, version 2.3.0 and later. To check the version, visit <domain>/<service>/version. Such as http://localhost:8080/ephox-hyperlinking/version.
```